patches/chrome-browser-ui-bookmarks-bookmark_tab_helper.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/ui/bookmarks/bookmark_tab_helper.cc b/chrome/browser/ui/bookmarks/bookmark_tab_helper.cc
+index f6ef61a0ac86ae55e8f9e80ece0758253af48520..888d656cf7080ebc32e90fc492e70a9b394d7026 100644
+--- a/chrome/browser/ui/bookmarks/bookmark_tab_helper.cc
++++ b/chrome/browser/ui/bookmarks/bookmark_tab_helper.cc
+@@ -67,6 +67,7 @@ bool BookmarkTabHelper::ShouldShowBookmarkBar() const {
+       !prefs->GetBoolean(bookmarks::prefs::kShowBookmarkBar))
+     return false;
+ 
++  return false;
```

Comment:
Was expecting to see something like `return !IsNTP(web_contents())` here. Is it safe to just return false here without further checks?

Reply:
I think this method is used for preventing bookmark bar visible or hidden opposite to the `kShowBookmarkBar` prefs value.

When `kShowBookmarkBar` is set to hidden, this method only returns true when active tab is NTP and bookmark bar isn't empty if all above conditions aren't met. So, we can hide bookmark bar on NTP by returning false in this case.

When `kShowBookmarkBar` is set to visible, this method isn't called except fullscreen condition.
According to the comment of `Browser::ShouldHideUIForFullscreen()`, browser controls UI can be visible on MacOS fullscreen mode by sliding down. When `Browser::ShouldHideUIForFullscreen()` returns true, this method will be called and we should also return false to hide bookmark bar.

So, I think returning false in here is safe.

Reply:
OK, got it.

Reply:
@emerick Thanks for quick review!